README.md

```diff
@@ -1,52 +1,124 @@
+#### New Project
+Our recomended way to start a new project with ApostropheCMS is to use the [Apostrophe CLI](https://github.com/apostrophecms/apostrophe-cli). Alternatively, you can simply clone this repo and `npm install`.
+##### Install the Apostrophe CLI
+ ```sh
+  npm install -g apostrophe-cli
+ ```
+##### Create a project
+ ```sh
+ apos create-project <shortname-without-spaces>
+ ```
 
-## Taking it Further
+`<shortname-without-spaces>` represents the "shortname" of the application. It will be used by default as a MongoDB database name and a basis for cookie names, etc. You will find this as the shortName property in app.js.
 
-Our documentation also covers advanced topics such as [managing permissions](https://docs.apostrophecms.org/apostrophe/tutorials/intermediate/permissions), [working with ApostropheCMS data](https://docs.apostrophecms.org/apostrophe/tutorials/intermediate/model-layer), [building forms](https://docs.apostrophecms.org/apostrophe/tutorials/intermediate/forms), and [hosting an ApostropheCMS site in production](https://docs.apostrophecms.org/apostrophe/tutorials/intermediate/deployment).
 
-Moreover, there's tons of great info in the ["How To"](https://docs.apostrophecms.org/apostrophe/tutorials/howtos) section of the docs, and it's always growing!
+## Extensions and Integrations
 
-## Extending the Core with Modules
+You can find a complete list of [extensions](https://apostrophecms.com/extensions) on our website, but here are a few highlights:
 
-ApostropheCMS is Node.js all the way through, which means it can be [extended](https://apostrophecms.com/extend) in all sorts of interesting ways. Here's a list of some of our favorite modules:
+  [apostrophe-headless](https://github.com/apostrophecms/apostrophe-headless) - Add REST APIs and power your React / Vue Native / etc. apps with a headless CMS.
+  [apostrophe-blog](https://github.com/apostrophecms/apostrophe-blog) - Everything you need to put a blog on your site.
+  [apostrophe-events](https://github.com/apostrophecms/apostrophe-events) - Manage upcoming events.
+  [apostrophe-workflow](https://github.com/apostrophecms/apostrophe-workflow) - Add powerful localization capabilities to your project.
+  [apostrophe-passport](https://github.com/apostrophecms/apostrophe-passport) - Authenticate via Twitter, Facebook, Github and more.
+  [apostrophe-saml](https://github.com/apostrophecms/apostrophe-saml) - Authenticate via Shibboleth, as well as corporate SAML environments like Salesforce.
+  [apostrophe-places](https://github.com/apostrophecms/apostrophe-places) - Manage geotagged content and display it on a map.
```

Comment:
Crush this with your new boot. Never tell anyone about apos-places, may it rest in power.

Reply:
Yep. apos-places is not good. We'll build a feature for geocoded pieces or geocoded fields eventually per past discussions, browser side code for maps always gets binned and rewritten because Projects Are Different.